src/Line.js

```diff
@@ -717,7 +722,8 @@ export function Line(opts, data) {
 
 				ctx.translate(-offset, -offset);
 			}
-		});
+        });
+        fire("drawGrid");
```

Comment:
let's move this up into `if (grid.show) {` above. it'd be weird if `drawGrid` fired when there's no grid to draw.

Reply:
Maybe `fire("drawAxis", axis)` would be more appropriate in that case? 

Can access grid from `axis.grid`

Reply:
yep, good catch.

Reply:
err, sorry. i meant `fire("drawGrid", i)`. the axes are not really "drawn" on the canvas (they're in the dom). so let's stick with canvas terminology.

Reply:
I changed this to `fire('drawGridAxis', a)` for each axis. If 1 or more axis are drawn, `drawGrid` will be fired. 

Allows hooking in directly after all axis have been drawn and before series is drawn.